sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -143,7 +143,8 @@ case class ExplainCommand(
     logicalPlan: LogicalPlan,
     extended: Boolean = false,
```

Comment:
instead of having so many boolean flags, can we use enum here?

Reply:
@cloud-fan There are quite a few callers of this method in test suites. Also, the way things are parsed, i have to put a if block in `SparkSqlParse.visitExplain` to populate the the enum value. If its ok, can take this as a small follow-up ?

Reply:
SGTM